libs/feature/catalog/src/lib/organisations/service/organisations-from-metadata.service.ts

```diff
@@ -58,7 +63,9 @@ export class OrganisationsFromMetadataService
             .filter((mail) => !!mail),
           recordCount: bucket.doc_count,
           description: null,
-          logoUrl: null,
+          logoUrl: bucket.logoUrl.buckets
+            .map((bucket) => bucket.key)
+            .filter((logo) => !!logo)[0],
```

Comment:
How the bucket key could be null ? To avoid this test, you can add `exclude` in the request.
Are they sorted by occurence ?

This should be enough ?
```suggestion
          logoUrl: bucket.logoUrl.buckets[0]?.key
```